features/LogFileFinder.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Log File Finder
+
+Scenario: Log File Finder should discover all the main output logs in the test folder
```

Comment:
James, in today's meeting, we disucssed RSpec tests.  I'm a little surprised to see Cucumber...what gives?